### ISSUE TITLE
Review and optimize help text and hints for both screens

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#138]
While working on design doc and posting issues at different times, I think I may have made some wording changes that should be reviewed en masse together after implementation.

We may want to tweak the wording when we see it all in context.

**Comment by emily-phet** [comment]
Agreed - we'll review all and refine once the next iteration is all in place.
